microservice/challenge-1/ReminderAppl/Dockerfile

```diff
@@ -0,0 +1,15 @@
+FROM openjdk:8-jdk-alpine
```

Comment:
Use a multistage dockerfile to build the artifact in one layer and serve it in another

Reply:
Multi-stage docker build applied